holmes/plugins/toolsets/aks.yaml

```diff
@@ -6,10 +6,6 @@ toolsets:
     prerequisites:
       - command: "az aks --help"
       - command: "kubectl version --client"
-      - env:
```

Comment:
Last time I introduce the requirement in https://github.com/robusta-dev/holmesgpt/pull/404/files is before we have the interactive mode, caching the cluster info was a problem.

Reply:
We should have reverted https://github.com/robusta-dev/holmesgpt/pull/404/files. It was merged so quick when it's title starts with DONTMERGE :)